8.x/getting-started/contributing.md

```diff
@@ -385,6 +385,115 @@ ___
 
 **10.** For every document that is written and placed in the documentation section of the Larammerce site, the members of the Larammerce team are obliged to read it.
 
+## PR review
+
+This section describes how to run, test and review your teammate's pull request in your local system.
+
+### Git
+
+`Git`*<sup>[4](#4)</sup>* is a `version control` or `source control` system which saves the source code and remembers the source code `changes` step by step as a stack. This stack is called `branch` and the changes are called `commit` or `revision`.
+
+If you order to cancel a commit, git creates a new commit which reverts the previous commit changes. So no commit is deleted from the git, but always the new commits are added. Git performs this task by adding and subtracting the lines. For example `+5 -3` in a given commit means that you added 5 lines to the source code and at the same time deleted 3 lines from it. Now if you order to delete this commit, git deletes the 5 new lines and adds the previous 3 lines and creates a new commit.
+
+In the git platform, the first branch is called `master` or `main`. A new lateral branch can be created and initialized from any given commit in the stack and, after several steps, merged into the master branch as a new commit.
+
+##### EXAMPLE
+
+Suppose you write the codes below:
+
+```bash
+#include<iostream>
+using namespace std;
+int main(){
+    cout<< "Hello!";
+    return 0;
+}
+```
+
+When you commit these codes, git makes a revision number 0 and assigns it a random string, like 223465:
```

Comment:
you can add some detail from the texts below:

To save these changes in Git, you create a commit. A commit is like a snapshot of your code at a particular point in time.
When you create a commit, Git takes all the changes you've made since the last commit and bundles them together into a new package which includes information about what files were changed, what lines were added or removed, and who made the changes. Git then assigns a unique identifier to this new commit in the form of a random string. This identifier allows Git to keep track of the commit and reference it later on.
each commit represents a new revision of your codebase.